content/management/changelog/index.mdx

```diff
@@ -38,6 +38,13 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**List Group Properties**](/management/configuration-api/v3.3/#list-group-properties) method had the `group_id` parameter renamed to `id`.
 - The [**Delete Group Properties**](/management/configuration-api/v3.3/#delete-group-properties) method had the `group_id` parameter renamed to `id`.
+- Properties configuration is now owned by **Client Id**, not by **License**, which resulted in the following changes:
+  - The **Register Properties** method was removed.
+  - The **List Registered Properties** method was removed.
+    - [**Register Property**](/management/configuration-api/v3.3/#register-property)
+    - [**Unregister Property**](/management/configuration-api/v3.3/#unregister-property)
+    - [**Publish Property**](/management/configuration-api/v3.3/#publish-property)
+    - [**List Properties**](/management/configuration-api/v3.3/#list-properties)
```

Comment:
Is spacing ok here? Looks like it is nested. Also, in the first two points, we used complete sentences, why no here?

Reply:
Good catch :) I Accidentally deleted one sentence. It was supposed to be a double-nested list.